tests/setup.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { test } from "@playwright/test";
+
+test("happy path setup", async ({ page }) => {
+  await page.goto("http://localhost:3000");
+
+  // setup coice
+  await page.getByLabel("Fresh Setup").click({ force: true }); // label is somehow covered
```

Comment:
What do you mean with the comment?

Reply:
For some reason the html/css setup is covering the label. But it's clearly working and works, so forcing it here seems valid 🤷